llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -121,23 +121,45 @@ static void writeToFile(std::string Filename, std::string Content) {
   OS.close();
 }
 
+// Describes scope covered by each entry in the module-kernel map populated by
+// the function below.
+enum KernelMapEntryScope {
+  Scope_PerKernel, // one entry per kernel
+  Scope_PerModule, // one entry per module
+  Scope_Global     // single entry in the map for all kernels
+};
+
 // Output parameter ResKernelModuleMap is a map containing groups of kernels
 // with same values of the sycl-module-id attribute.
 // The function fills ResKernelModuleMap using input module M.
```

Comment:
I think we should rewrite this comment somehow.

Reply:
the comment has been invalid since addition of per-kernel split before my changes.
But OK, I'll go ahead and change the comment